compiler/x/codegen/X86Ops.ins

```diff
@@ -2470,6 +2474,10 @@ INSTRUCTION(PCMPEQBRegReg, pcmpeqb,
             BINARY(VEX_L128, VEX_vReg_, PREFIX_66, REX__, ESCAPE_0F__, 0x74, 0, ModRM_RM__, Immediate_0),
             PROPERTY0(IA32OpProp_SourceRegisterInModRM),
             PROPERTY1(IA32OpProp1_XMMSource | IA32OpProp1_XMMTarget)),
+INSTRUCTION(VPCMPEQBRegMem, vpcmpeqb,
+            BINARY(VEX_L256, VEX_vReg_, PREFIX_66, REX__, ESCAPE_0F__, 0x74, 0, ModRM_RM__, Immediate_0),
+            PROPERTY0(IA32OpProp1_SourceIsMemRef),
+            PROPERTY1(IA32OpProp1_XMMTarget)),
```

Comment:
Missing XMMSource property?

Reply:
This is RegMem version of instruction vpcmpeqb, so that it has `IA32OpProp1_SourceIsMemRef` but should not have the XMMSource one.

Reply:
But it's a good reminder that we probably will need the RegReg version as well. I've added the RegReg version along with this RegMem version.